ui/ducks/confirm-alerts/confirm-alerts.ts

```diff
@@ -0,0 +1,116 @@
+import { Severity } from '../../helpers/constants/design-system';
+
+export type Alert = {
+  key: string;
+  field?: string;
+  severity: Severity.Danger | Severity.Warning | Severity.Info;
+  message: string;
+  reason?: string;
+  alertDetails?: string[];
+};
+
+export type ConfirmAlertsState = {
```

Comment:
JSDoc? Especially to clarify what the alerts are keyed against and what `ownerId` is.